public/js/Date.prototype.extensions.js

```diff
@@ -11,20 +11,20 @@ Date.prototype.createUTCDate = function () {
   );
 };
 Date.prototype.displayDMY = function () {
-  const M = [
-    'January',
-    'February',
-    'March',
-    'April',
-    'May',
-    'June',
-    'July',
-    'August',
-    'September',
-    'October',
-    'November',
-    'December',
-  ];
+  // const M = [
```

Comment:
I commented out all unused code/variables I could find without breaking anything. I'm not going to fully remove it though for now